test/test_helper.rb

```diff
@@ -21,89 +20,89 @@ def teardown
-    Item.create(name: "gnar possum",
-                description: "a snowboard for shredding gnar pow",
+    Item.create(name: 'gnar possum',
+                description: 'a snowboard for shredding gnar pow',
                 price: 1000)
-    Item.create(name: "gwar possum",
-                description: "a snowboard for gwar concerts",
+    Item.create(name: 'gwar possum',
+                description: 'a snowboard for gwar concerts',
                 price: 15)
   end
 
   def create_items_and_order
-    @category     = Category.create(title: "Snowboards")
-    @category_two = Category.create(title: "Apparel")
-    item_1 = Item.create(name: "gnar possum",
-                description: "a snowboard for shredding gnar pow",
-                price: 1000,
-                category_id: @category.id)
-    item_2 = Item.create(name: "gwar possum",
-                description: "a snowboard for gwar concerts",
-                price: 15,
-                category_id: @category.id)
-    item_3 = Item.create(name: "Hoody",
-                description: "Keeps you slightly warm",
-                price: 80,
-                category_id: @category_two.id)
-    Order.create(current_status: "completed")
+    @category     = Category.create(title: 'Snowboards')
+    @category_two = Category.create(title: 'Apparel')
+    item_1 = Item.create(name: 'gnar possum',
+                         description: 'a snowboard for shredding gnar pow',
+                         price: 1000,
+                         category_id: @category.id)
+    item_2 = Item.create(name: 'gwar possum',
+                         description: 'a snowboard for gwar concerts',
+                         price: 15,
+                         category_id: @category.id)
+    item_3 = Item.create(name: 'Hoody',
+                         description: 'Keeps you slightly warm',
+                         price: 80,
+                         category_id: @category_two.id)
+    Order.create(current_status: 'completed')
   end
 
   def create_user
-    User.create(username: "Matt", password: "gnargnar")
+    User.create(username: 'Matt', password: 'gnargnar')
   end
 
   def login_a_user
     create_user
     visit login_path
 
-    within(".login_form") do
-      fill_in "Username", with: "Matt"
-      fill_in "Password", with: "gnargnar"
-      click_button "Login"
+    within('.login_form') do
+      fill_in 'Username', with: 'Matt'
+      fill_in 'Password', with: 'gnargnar'
+      click_button 'Login'
     end
   end
 
   def add_items_to_cart
-    item_1 = Item.create(name: "gnar possum",
-                         description: "a snowboard for shredding gnar pow",
+    item_1 = Item.create(name: 'gnar possum',
+                         description: 'a snowboard for shredding gnar pow',
                          price: 1000)
-    item_2 = Item.create(name: "gnar possum",
-                         description: "a snowboard for shredding gnar pow",
+    item_2 = Item.create(name: 'gnar possum',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.